sandbox/apparmor/apparmor.go

```diff
@@ -663,7 +663,7 @@ func probeKernelFeatures() ([]string, error) {
 			features = append(features, "policy:permstable32:"+feat)
 		}
 	}
-	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "notify", "user")); err != nil {
+	if data, err := os.ReadFile(filepath.Join(rootPath, featuresSysPath, "policy", "notify", "user")); err == nil {
 		// XXX: there's no feature added for policy:notify:user, since user is
```

Comment:
Nitpick: I would argue this is confusing.

`no feature added for policy:notify:user` then VS `"policy:notify:user:"+feat`

This comes from place of naively populating the representation structure from path elements which is not the case anymore.. Perhaps clarify to explain that "policy:notify:user" should not exist on its own. Perhaps even test for this?

Reply:
Is the comment confusing or the fact that no `policy:notify:user` value appears in the kernel features?